app/assets/stylesheets/components/shared/c-card.scss

```diff
@@ -1,14 +1,18 @@
 
 .c-card {
+  display: flex;
   position: relative;
-  margin: 0 0 20px;
+  flex-direction: column;
+  height: $card-height;
```

Comment:
I'm a bit worried this will give us issues with the responsiveness or if the content of a card is longer than expected. Could we have a `min-height` instead?